dev/wlp.lib.extract_fat/fat/src/wlp/lib/extract/PackageRunnableTest.java

```diff
@@ -49,24 +49,30 @@
     /*
      * return env as array and add WLP_JAR_EXTRACT_DIR=extractDirectory
      */
-    private static String[] runEnv(String extractDirectory) {
+    private static String[] runEnv(String extractDirectory, boolean useDummyUserDir) {
```

Comment:
Some comments on when to useDummyUserDir might be helpful. If we have this Boolean I'm wondering why not have a test where we test the true scenario as well?

Reply:
Thanks for taking a look.

I added some comments in my updated (and rebased) PR .

I actually did refactor the `testRunnableJar` test so the net result is both true/false boolean values are passed through this method (both are passed to the `executeTheJar` method as well).

Also this test class has some pretty heavy-weight methods with forking processes and the cleanup involved (including a sleep for 30 seconds)... which makes me think this isn't going to be one where we add too many more different test methods to.